doc/rules/whitespace/statement_indentation.rst

```diff
@@ -35,8 +96,14 @@ The rule is part of the following rule sets:
 - `@PER-CS2.0 <./../../ruleSets/PER-CS2.0.rst>`_
 - `@PSR2 <./../../ruleSets/PSR2.rst>`_
 - `@PSR12 <./../../ruleSets/PSR12.rst>`_
-- `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_
-- `@Symfony <./../../ruleSets/Symfony.rst>`_
+- `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_ with config:
+
+  ``['stick_comment_to_next_continuous_control_statement' => true]``
+
+- `@Symfony <./../../ruleSets/Symfony.rst>`_ with config:
+
+  ``['stick_comment_to_next_continuous_control_statement' => true]``
```

Comment:
this turned out to be too agressive - https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7497 - and should therefore be not by default in any ruleset

Reply:
Why _any_ ruleset? If Symfony's standard is like this, then it's OK to have it enabled. If you don't like this and you're using Symfony or PhpCsFixer standard as a base for your ruleset, then you need to re-configure this particular rule 🙂.

Reply:
No. The Symfony or PhpCsFixer standards are too aggresive in cases where comments are at the places mentioned in the issue, search https://github.com/roundcube/roundcubemail codebase for "7497", there are dozens places.

Reply:
So don't use these rulesets 🤷‍♂️? These are opinionated sets, you either accept how they work or prepare your own configuration. And just to remind, I don't like this behaviour at all because I believe it's impossible to determine the intent (where the comment belongs to) and from a logical point of view such comments should be intended just like everything inside the braces. But it was done on purpose to move it one step left, and now it can be configurable - leaving the same behaviour as default. It's the decision of rulesets' owners, not something you can argue with 🙂.

Reply:
The PHP CS Fixer is not reformatter, thus it should reformat to one, current?, indent, but allow the other valid/meaningful one too. I would say that would be better than hard/project wide configuration option.

Reply:
Reformatting is part of Fixer responsibility. Actually, quite a huge part of it - anything related to whitespaces or phpdocs can be understood as reformatting.

With previous complain that the change was too aggressive, we disable it by default for rule itself and allow everyone to opt-in, if they want to use it.

Same time, any community creating their standard has a freedom to decide where the comments should be placed. And it is that implemented rules are following existing standard, not that standard is adjusted to created rules. With that, if someone was already following Sf standards, they shall already have following each rules, even those without existing Fixer rules.

Any standard/rule can be called "too aggressive" by those who do not want to follow given rule. Happily, you can not only disable the rule now, but also configure it to your needs.